hmt_escrow/eth_bridge.py

```diff
@@ -28,9 +28,10 @@
 ])
 
 # See more details about the eth-kvstore here: https://github.com/hCaptcha/eth-kvstore
-KVSTORE_CONTRACT = Web3.toChecksumAddress(
-    os.getenv("KVSTORE_CONTRACT",
-              "0xbcF8274FAb0cbeD0099B2cAFe862035a6217Bf44"))
+# KVSTORE_CONTRACT = Web3.toChecksumAddress(
+#      os.getenv("KVSTORE_CONTRACT", "0xbcF8274FAb0cbeD0099B2cAFe862035a6217Bf44"))
+
+KVSTORE_CONTRACT = Web3.toChecksumAddress("0xbcF8274FAb0cbeD0099B2cAFe862035a6217Bf44")
```

Comment:
You'll need to read the `KVSTORE_CONTRACT` from the env, `0xbcF8274FAb0cbeD0099B2cAFe862035a6217Bf44` is a default that's not actually working.